### PR TITLE
GDB-10391 - Render menu labels faster

### PR DESCRIPTION
**Pull request by DesiBorisova**
## What?
The "Import" and "SPARQL" main menu labels will render faster on window resize.

## Why?
The two labels would load slower than the rest during resize.

## How?
I changed the ng-if to ng-show and added a debounce function to the function called.

### CODE REVIEW

File: src/js/angular/controllers.js [hunk 1]
```diff
@@ -716,6 +719,40 @@ function mainCtrl($scope, $menuItems, $jwtAuth, $http, toastr, $location, $repos
...
+     * Creates a debounced function that delays invoking the provided function until after a specified wait time has
+     * elapsed since the last time the debounced function was invoked. Optionally, the function can be invoked immediately
+     * and then wait for the specified time before allowing the next invocation.
+     * @param {Function} func - The function to debounce.
+     * @param {number} wait - The number of milliseconds to delay.
+     * @param {boolean} [callImmediately=false] - If true, the function will be invoked immediately, then wait for the specified time
+     *                                           before allowing the next invocation.
+     * @return {Function} - Returns the new debounced function.
+     */
+    function debounce(func, wait, callImmediately) {
```
Comment (svilenvelikov) [inline src/js/angular/controllers.js:732]: Two notes here. First, why this is here and not in some utility and second, why not just using the lodash's debounce function?

Comment (DesiBorisova) [reply src/js/angular/controllers.js:732]: The lodash debounce doesn't always work so well in this case. The labels still sometimes load slower. I will move this to a utility class instead.

Comment (svilenvelikov) [reply src/js/angular/controllers.js]: Well, how that can be?! :) Did you try tweaking the debounce parameters? Try passing `{trailing: false}`.

File: src/js/angular/controllers.js [hunk 2]
```diff
@@ -23,6 +23,7 @@ import './guides/directives';
 import {GUIDE_PAUSE} from './guides/tour-lib-services/shepherd.service';
 import 'angular-pageslide-directive/dist/angular-pageslide-directive';
 import 'angularjs-slider/dist/rzslider.min';
+import _ from "lodash";
```
Comment (svilenvelikov) [inline src/js/angular/controllers.js:26]: import only the debounce function and not the entire thing please

Comment (DesiBorisova) [reply src/js/angular/controllers.js]: Done :smile:

**Review by svilenvelikov — APPROVED** [review]

**Pull request merged**